src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceNewBookingTest.kt

```diff
@@ -503,5 +863,290 @@ class OffenderResourceNewBookingTest : ResourceTest() {
+        assertThat(getCurrentIEP(offenderNo))
+          .extracting(PrivilegeSummary::getIepLevel)
+          .isEqualTo("Enhanced")
+
+        webTestClient.post()
+          .uri("/api/offenders/{offenderNo}/booking", offenderNo)
+          .headers(
+            setAuthorisation(
+              listOf("ROLE_BOOKING_CREATE")
+            )
+          )
+          .header("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+          .bodyValue(
+            """
+            {
+               "prisonId": "MDI", 
+               "fromLocationId": "COURT1", 
+               "movementReasonCode": "24", 
+               "imprisonmentStatus": "CUR_ORA" 
+            }
+            """.trimIndent()
+          )
+          .accept(MediaType.APPLICATION_JSON)
+          .exchange()
+          .expectStatus().isOk
+
+        assertThat(getCurrentIEP(offenderNo))
+          .extracting(PrivilegeSummary::getIepLevel)
+          .isEqualTo("Entry")
+      }
+
+      @Test
+      internal fun `will create admission case note`() {
+        val bookingId = webTestClient.post()
+          .uri("/api/offenders/{offenderNo}/booking", offenderNo)
+          .headers(
+            setAuthorisation(
+              listOf("ROLE_BOOKING_CREATE")
+            )
+          )
+          .header("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+          .bodyValue(
+            """
+            {
+               "prisonId": "MDI", 
+               "fromLocationId": "COURT1", 
+               "movementReasonCode": "24", 
+               "imprisonmentStatus": "CUR_ORA"
+            }
+            """.trimIndent()
+          )
+          .accept(MediaType.APPLICATION_JSON)
+          .exchange()
+          .expectStatus().isOk
+          .returnResult(InmateDetail::class.java)
+          .responseBody.blockFirst()!!.bookingId
+
+        assertThat(getCaseNotes(bookingId))
+          .extracting(CaseNote::getType, CaseNote::getSubType, CaseNote::getText)
+          .contains(
+            tuple(
+              "TRANSFER",
+              "FROMTOL",
+              "Offender admitted to MOORLAND for reason: Recall From Intermittent Custody from Court 1."
+            )
+          )
+      }
+    }
+  }
+
+  fun createActiveBooking(prisonId: String = "MDI"): String = OffenderBuilder().withBooking(
+    OffenderBookingBuilder(
+      prisonId = prisonId,
+    )
+  ).save(
+    webTestClient = webTestClient,
+    jwtAuthenticationHelper = jwtAuthenticationHelper,
+    dataLoader = dataLoader
+  ).offenderNo
+
+  fun createInactiveBooking(iepLevel: String = "ENH"): String = OffenderBuilder().withBooking(
+    OffenderBookingBuilder(
+      prisonId = "MDI",
+      released = true
+    ).withIEPLevel(iepLevel)
+  ).save(
+    webTestClient = webTestClient,
+    jwtAuthenticationHelper = jwtAuthenticationHelper,
+    dataLoader = dataLoader
+  ).offenderNo
+
+  private fun getMovements(bookingId: Long) = externalMovementRepository.findAllByOffenderBooking_BookingId(bookingId)
```

Comment:
A load of these methods are duplicated in the transfer test; might see how they can be shared